tests/unit/test_config.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+def test_output_format(test_client, test_sample_dataset):
+    import random
+
+    # JSON output
+    test_client.output_format = "json"
+    results = test_client.services.search.vector(
+        test_sample_dataset,
+        multivector_query=[
+            {
+                "vector": [random.randint(0, 1000) for _ in range(100)],
+                "fields": ["sample_1_vector_"]
+            }
+        ]
+    )
+    assert isinstance(results, dict)
+
+    # Content output
+    test_client.output_format = "content"
```

Comment:
Separate this into a separate test. Try to only have 1 assert each test.

Reply:
Look into @pytest.mark.parametrize if you want to make it easier for yourself :)

Reply:
Done!
Good learn - Thanks